mcan/src/bus.rs

```diff
@@ -575,6 +575,16 @@ impl<'a, Id: mcan_core::CanId, D: mcan_core::Dependencies<Id>, C: Capacities>
         Ok(can)
     }
 
+    /// Locks the configuration and enters initialization mode.
+    pub fn finalize_initialized(mut self) -> Result<Can<'a, Id, D, C>, ConfigurationError> {
```

Comment:
Just to raise the thought.

Do we gain something in particular by having both `finalize` and `finalize_initialized`.
If the user then wanted to go to `Normal mode` after init is as simple as running `aux.operational_mode()`

I'm all for choice, so it's not really a big deal for me. Just thought about it.

Thoughts?

Reply:
I agree that this is worth considering. With this approach, I guess it guides the user into making the choice that is most probably correct for them, in the same vein as the other method simply being called `finalize()` and not `finalize_normal_mode()` or something to that effect.

On the other hand, using this library expects the user to be quite aware of how MCAN works, which doesn't put unreasonable expectations on the user if they need to run `aux.operational_mode()` after running this method in order to enter `Normal mode`.

Reply:
Agree, let's keep it as is :+1: